server_admin/topics/login-settings/remember-me.adoc

```diff
@@ -1,21 +1,17 @@
 
 === Remember Me
 
-If a logged in user closes their browser, their session is destroyed and they will have to log in again.  You can set things
-up so that if a user checks a _remember me_ checkbox, they will remain logged in even if the browser is closed.  This basically
-turns the login cookie from a session-only cookie to a persistence cookie.
-
-To enable this feature go to `Realm Settings` left menu item and click on the `Login` tab and turn on the `Remember Me` switch:
+A logged-in user closing their browser destroys their session, and they must log in again. You can set {project_name} to keep the user's login session open if they click the _Remember Me_ checkbox when they log in. This action turns the login cookie from a session-only cookie to a persistence cookie.
```

Comment:
change "they" to "that user"  -- TWICE
change "when they log in" to "upon login." to avoid a third "that user"